src/irclog2html/irclog2html.py

```diff
@@ -160,6 +162,13 @@ def __iter__(self):
             else:
                 time = None
 
+            if time is None:
+                m = self.TIMESTAMP_REGEXP.match(line)
+                if m:
+                    time = datetime.datetime.fromtimestamp(
+                        int(m.group(1))).isoformat()
+                    line = line[len(m.group(0)):]
```

Comment:
The Travis failure looks to be caused by timezones.  Now I'm wondering if this should be `utcfromtimestamp()`, or if I should change my test harness to set `TZ=UTC`.

Reply:
Indeed it makes sense to parse timestamp as UTC.